### ISSUE TITLE
Check whether Deft is loaded

### ISSUE THREAD

**Issue by EFLS** [EFLS/zetteldeft#59]
Deft needs to be loaded for any Zetteldeft search function to work, but this is never checked.

So when a user calls, for example `zetteldeft-follow-link` right after launching Emacs, it won't work.

Not sure how or where to check for this.

**Comment by stampf** [comment]
Yes, I have the same issue. So I bound F8 to Deft mode (which opens in my zettelkasten), and from there I can open a zettelnote and everything's fine.
But with the emacs dashboard for instance, when I open a previously opened note right after emacs launched, Deft isn't loaded and nothing works.
